api/views.py

```diff
@@ -10,7 +10,15 @@
 
 class ManifestViewSet(viewsets.ReadOnlyModelViewSet):
     """
-    API endpoint for returning a list of of manifests.
+    list: API endpoint for returning a list of of manifests.
+
+    For each data source file, there is a corresponding manifest containing an
```

Comment:
heads up, this all falls apart if you use tabs instead of spaces. the retrieve section is viewed as a continuation of the list section